cypress/support/commands.js

```diff
@@ -415,17 +415,9 @@ Cypress.Commands.add('showHiddenFiles', () => {
 	cy.get('.modal-container__close').click()
 })
 
-Cypress.Commands.add('createDescription', () => {
-	const url = '**/remote.php/dav/files/**'
-	cy.intercept({ method: 'PUT', url })
-		.as('addDescription')
-
-	cy.get('[data-cy-files-list] tr[data-cy-files-list-row-name="Readme.md"]').should('not.exist')
-	cy.get('.files-controls').first().within(() => {
-		cy.get('.button.new').click()
-		cy.get('.newFileMenu a.menuitem[data-action="rich-workspace-init"]').click()
-	})
-	cy.wait('@addDescription')
```

Comment:
We still need to bring this back with the new vue files app similar to https://github.com/nextcloud/server/pull/40854/files#diff-7689f04d5c686a19ee55422032850ea26799cf003b0730162ecc29b01a5cc91bR77-R92

Let's track this in a separate ticket as I think bringing back the tests is a good intermediate step.

Reply:
I agree, I will open a new issue.